webpack.main.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+
+const config = {
+  mode: 'production',
+  entry: './src/index.ts',
+  target: "electron-main",
+  devtool: "inline-source-map",
```

Comment:
that's the reason we only got 10% out of it..

Reply:
Look at the end of the file